src/java.base/share/classes/java/text/DecimalFormat.java

```diff
@@ -38,6 +38,9 @@
 
 package java.text;
 
+import sun.util.locale.provider.LocaleProviderAdapter;
+import sun.util.locale.provider.ResourceBundleBasedAdapter;
+
```

Comment:
Internal packages typically follow public packages.

Reply:
Fixed. (IntelliJ is adamant on having it that way, and even after I previously reverted it, it seems like it slipped through again. Oddly, it never used to do this, need to look at the settings)